### PR TITLE
Add tests moving the robot

### PR DESCRIPTION
**Pull request by fmauch**
This is basically #33 for Iron (and Humble)

### CODE REVIEW

**Comment by fmauch** [comment]
@Mergifyio backport humble

File: ur_simulation_gazebo/test/test_common.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+import logging
+import time
+
+import rclpy
+from rclpy.action import ActionClient
+
+from controller_manager_msgs.srv import ListControllers
+
+TIMEOUT_WAIT_SERVICE = 10
+TIMEOUT_WAIT_SERVICE_INITIAL = 120  # If we download the docker image simultaneously to the tests, it can take quite some time until the dashboard server is reachable and usable.
```
Comment (VinDp) [inline ur_simulation_gazebo/test/test_common.py]:
```suggestion
TIMEOUT_WAIT_SERVICE_INITIAL = 120
```

File: ur_simulation_gazebo/test/test_gazebo.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+# Copyright 2023, FZI Forschungszentrum Informatik
```
Comment (VinDp) [inline ur_simulation_gazebo/test/test_gazebo.py]:
```suggestion
# Copyright 2024, FZI Forschungszentrum Informatik
```

File: ur_simulation_gazebo/test/test_common.py [hunk 3]
```diff
@@ -0,0 +1,180 @@
+# Copyright 2023, FZI Forschungszentrum Informatik
```
Comment (VinDp) [inline ur_simulation_gazebo/test/test_common.py]:
```suggestion
# Copyright 2024, FZI Forschungszentrum Informatik
```

**Review by VinDp — APPROVED** [review]
Everything looks and works good: I have only very small suggestions.

**Pull request merged**